lib/c_glib/test/testthrifttestclient.cpp

```diff
@@ -112,8 +112,8 @@ class TestHandler : public ThriftTestIf {
     out = thing;
   }
 
-  std::string testUuid(const std::string thing) override {
-    cout << "[C -> C++] testUuid(\"" << std::hex << thing << "\")" << '\n';
+  apache::thrift::TUuid testUuid(const apache::thrift::TUuid thing) override {
```

Comment:
If you use const ref on the parameter and return value instead, it avoids copies.

Reply:
Unfortunately thirft generates the interface with a copy. Perhaps it is something I missed when I added the new type because I see the other functions uses const ref on some arguments

Reply:
@Jens-G Should I create a ticket for this and fix asap?
It would change how the method is generated if I change it to be a reference:
It appears to change to
```
virtual void testUuid(apache::thrift::TUuid& _return, const apache::thrift::TUuid& thing) = 0;
```
but I can't work on this more at this moment, probably only tonight again

Reply:
@jeking3  I made a new PR related to this, would you be so kind to look at it please:
https://github.com/apache/thrift/pull/3038